Lib/glyphsLib/builder/font.py

```diff
@@ -51,6 +51,9 @@ def to_ufo_font_attributes(self, family_name):
     # note = font.note
     glyph_order = list(glyph.name for glyph in font.glyphs)
 
+    if not self.store_editor_state:
+        font.DisplayStrings = None
```

Comment:
Instead of changing the font in place, could this be done at the stage where the DisplayStrings are put in the lib? 

I think it's weird if you do `font = GSFont(); doc = to_designspace(font)` and after that, `font` is modified. The whole glyphs->ufo process should only read from it.

I think this should be handled in custom_params.py, like that somehow:
```py
class DisplayStringsParamHandler(MiscParamHandler):
    def __init__(self):
        super(...).__init__(glyphs_name="DisplayStrings")

    def to_ufo(self, builder, glyphs, ufo):
        if builder.store_editor_state:
            super(...).to_ufo(...)

register(DisplayStringsParamHandler())
```

but that would mean changing the signature of `ParamHandler#to_ufo` and `#to_glyphs` to take the builder as an argument and be able to read options from it

Reply:
Gah. I looked at that first and thought "nah!". Maybe I could just remove it from the UFO lib afterwards?